src/cases/containers/case-home/case-home.component.ts

```diff
@@ -1,10 +1,113 @@
 import { ExUITitleService } from 'src/app/shared/services/exui-title.service';
-import { Component } from '@angular/core';
+import { Component, OnInit, OnDestroy, ViewEncapsulation } from '@angular/core';
+import { Subject, Subscription } from 'rxjs';
+import { AlertService, NavigationNotifierService, NavigationOrigin, HttpError } from '@hmcts/ccd-case-ui-toolkit';
+import { Store } from '@ngrx/store';
+import * as fromRoot from '../../../app/store';
+import * as fromFeature from '../../store';
+import { GoActionParams } from 'src/cases/models/go-action-params.model';
 
 @Component({
   selector: 'exui-case-home',
-  templateUrl: 'case-home.component.html'
+  templateUrl: 'case-home.component.html',
+  styleUrls: ['case-home.component.scss']
 })
-export class CaseHomeComponent {
-  constructor(private titleService: ExUITitleService) {}
+export class CaseHomeComponent implements OnInit, OnDestroy {
+
+  public static readonly CASE_CREATED_MSG = 'The case has been created successfully';
+  public static readonly DRAFT_DELETED_MSG = 'The draft has been successfully deleted';
+
+  callbackErrorsSubject: Subject<any> = new Subject();
+  navigationSubscription: Subscription;
+
+  constructor(
+    private titleService: ExUITitleService,
+    private alertService: AlertService,
+    private navigationNotifier: NavigationNotifierService,
+    private store: Store<fromFeature.State>,
+  ) { }
+
+  ngOnInit(): void {
+    this.navigationSubscription = this.navigationNotifier.navigation.subscribe(navigation => {
+      if (navigation.action) {
+        this.actionDispatcher(this.paramHandler(navigation));
```

Comment:
this should move trigger effects

Reply:
I'm not quite clear what you mean with this. The component only needs to trigger the `Go` action. The parameters and callbacks/error-handlers are prepared and passed on to the action. I don't understand where or why there is a need for an effect.

Reply:
yeah fine but is a side effect and needs to be managed in the effect... so is just about to move this code to the effects